### PR TITLE
Add php-cs-fixer dependency and default config

### LINKED ISSUE #388: Add php-cs-fixer and default configuration
[PHP Coding Standards Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) would be a great way to worry less about format/indenting guide lines.

Each fixer can be explicitly included or excluded so it should be no problem to reflect the current practices.
You can define project defaults in a file `.php_cs` which will be used when running it with `./vendor/bin/php-cs-fixer fix`.

I'd recommend using the PSR2 level as a starting point and then including any extra fixers for formatting (e.g. no spaces when concatenating).

### PR DESCRIPTION
**Pull request by nochso**
Closes #388

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
         "mikey179/vfsStream": "^1.5",
         "phpunit/phpunit": "~4.0",
         "symfony/validator": "~2.6.9",
-        "zendframework/zend-validator": "~2.3"
+        "zendframework/zend-validator": "~2.3",
+        "fabpot/php-cs-fixer": "^1.10"
```
Comment (henriquemoody) [inline composer.json]: This is better as suggested than request-dev

**Comment by henriquemoody** [comment]
LGTM, can you squash these commits at once?

File: .php_cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<?php
+
+$finder = Symfony\CS\Finder\DefaultFinder::create()
```
Comment (henriquemoody) [inline .php_cs]: Can you add *.php as a valid extension too?

Comment (nochso) [inline .php_cs]: Will do.

I just remembered this should probably be noted in CONTRIBUTING. Add it to the intro or mention it in a new section after unit testing?

Comment (henriquemoody) [inline .php_cs]: That would be awesome!

Comment (henriquemoody) [inline .php_cs]: Ad *.phpt as well, please

Comment (nochso) [inline .php_cs]: Okay, now filtering *.php and *.php. Also added a new section in CONTRIBUTING.md about coding standards.

**Comment by henriquemoody** [comment]
Thanks, @nochso! :panda_face:

**Comment by nochso** [comment]
Pandas make my day :+1:

**Pull request merged**